test/api.js

```diff
@@ -1,9 +1,9 @@
-const standard = require('../')
-const test = require('tape')
+import test from 'tape'
+import linter from '../index.js'
```

Comment:
Why are you renaming from `standard` to `linter` ?
I'm not against this change, just wanting to know if there's a specific reason.

Reply:
I don't know. can revert.